BrightFutures/ExecutionContext.swift

```diff
@@ -22,12 +22,18 @@
 
 import Foundation
 
+/**
+ *  ExecutionContext is a protocol that defines a context in which tasks can be executed.
```

Comment:
You're using a different syntax here than in most of the other cases. It should be consistent.